Automattic/woocommerce-payments
# Allow subscription processing via WooPay

Fixes 296-gh-Automattic/woopay

Project Thread: pdpOw2-Wt-p2

#### Changes proposed in this Pull Request

This PR removes the feature flag on subscription processing via WooPay, so that customers can use WooPay to complete subscription purchases.

#### Testing instructions

See project thread (pdpOw2-Wt-p2) for full details on tests conducted for this subscriptions integration.

For testing this PR (removal of the feature flag), test:
- With only a subscription product to your cart, ensure that WooPay is available on checkout
- With only a non-subscription product in your cart, ensure that WooPay is available on checkout
- Disable WooPay in merchant settings, ensure that WooPay is **not** available on checkout
- Enable WooPay, but then mark your site as ineligible for WooPay, ensure that WooPay is **not** available on checkout

-------------------


**Post merge**
